### ISSUE TITLE
Two json files are the same

### ISSUE THREAD

**Issue by KimHe** [MartinThoma/propy3#8]
Martin, please check the following two files:

- 'grantham*.json' 

and 

- 'schneider-wrede*.json'

They are the same! Did you spoil them during file-renaming

Grantham scores should be integers between [5, 215]

**Comment by MartinThoma** [comment]
oO ... It could very well be that I introduced bugs while porting the project to Python 3 + improving the code in various ways :see_no_evil: 

As I mentioned before, I will only put a minimum amount of effort in this. I'm happy to accept PRs / introduce the fix if you show me exactly how things should be

**Comment by jowch** [comment]
I figured I'd take a look at this too since I have a copy of the original propy lying around. My copy of it is in a private repo so I can't link the code, but I can provide a snippet. The location in the Python 2 code where it is referenced looks like this:
```python
## Distance is the Schneider-Wrede physicochemical distance matrix used by Chou et. al. 
_Distance1={"GW":0.923, "GV":0.464, "GT":0.272, "GS":0.158, "GR":1.0, "GQ":0.467, "GP":0.323, "GY":0.728, ...
## Distance is the Grantham chemical distance matrix used by Grantham et. al. 
_Distance2={"GW":0.923, "GV":0.464, "GT":0.272, "GS":0.158, "GR":1.0, "GQ":0.467, "GP":0.323, "GY":0.728, ...
```
I've truncated the lines for viewing but they are identical in the file and identical to the JSON files. This is probably why the JSON files are the same. @KimHe, do you know whether they should be different? I'm not knowledgable enough to say one way or another.

**Comment by jhahnfeld** [comment]
That the two json files are the same is indeed a bug.
> The Grantham scores should be integers between [5, 215]

Source: [Grantham-paper](https://doi.org/10.1126/science.185.4154.862) table 2 (sadly hidden behind a paywall)

The Schneider-Wrede scores are correct ([paper](https://www.sciencedirect.com/science/article/pii/S0006349594807829) table 1).
The bug already existed in the original propy implementation.

The correct matrices are available in the R package [protr](https://github.com/nanxstats/protr):
[Schneider-Wrede](https://github.com/nanxstats/protr/blob/master/inst/sysdata/Schneider-Wrede.csv)
[Grantham](https://github.com/nanxstats/protr/blob/master/inst/sysdata/Grantham.csv)

**Comment by MartinThoma** [comment]
Fixed by the merged pr

**Comment by jhahnfeld** [comment]
Hi Martin,

we forgot to make a patch release with the corrected values. :sweat: 
Could you please update the PyPi version.
I would update the Bioconda version afterwards.

Julian

**Comment by MartinThoma** [comment]
I've just released 1.1.1 on pypi. Thank you for the reminder 🙏
